src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -8,13 +8,66 @@ categories:
   - mobile
 ---
 
-The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) Navigation support for Sentry.
+The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) support for the following interactions:
+* User interactions like clicks, swipe and scroll gestures
+* Jetpack Compose navigation
+
+## User Interactions
+This feature is disabled by default, but you can enable it the following ways:
+
+### Using `AndroidManifest.xml`
+```xml {filename:AndroidManifest.xml}
+<application>
+    <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />
+</application>
+```
+
+### Using `SentryOptions`
+If you initialize the SDK [manually as mentioned here](/platforms/android/configuration/manual-init/) you can enable user interactions like this:
+```kotlin {filename:MyApplication.kt}
+SentryAndroid.init(this) { options ->
+    // ...
+    options.isEnableUserInteractionTracing = true
+    options.isEnableUserInteractionBreadcrumbs = true
+}
+```
+
+The Sentry SDK utilizes the built-in `Modifier.testTag("<tag>")` of a Composable to determine it's identifier. Here's an example:
+
+```kotlin
+import androidx.compose.ui.platform.testTag
+@Composable
+fun LoginScreen() {
+  Column {
+    // ...
+    Button(
+        modifier = Modifier.testTag("button_login"),
+        onClick = { TODO() }) {
+        Text(text = "Login")
+    }
+  }
+}
+```
+
+<Note>
+
+If a `@Composable` doesn't have a `testTag` modifier applied, both breadcrumbs and transactions won't be captured because they can't be uniquely identified.
+
+</Note>
+
+<Note>
+
+If you want to customize the recorded breadcrumbs / transactions you can skip [to section below](#customize-the-recorded-breadcrumbtransaction).
+
+</Note>
+
+## Jetpack Compose Navigation
 
-On this page, we get you up and running with Sentry's Compose Navigation Integration, so that it will automatically add a breadcrumb and start a transaction for each navigation event.
+On this section, we get you up and running with Sentry's Compose Navigation Integration, so that it will automatically add a breadcrumb and start a transaction for each navigation event.
```

Comment:
```suggestion
In this section, we get you up and running with Sentry's Compose Navigation Integration, so that it will automatically add a breadcrumb and start a transaction for each navigation event.
```